app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -443,230 +535,196 @@ $search-icon-size: 20px;
-    padding: govuk-spacing(6);
+  @include govuk-media-query($from: 360px) {
+    right: 0;
   }
 
   @include govuk-media-query($from: "desktop") {
-    @include govuk-font($size: 16, $weight: false, $line-height: 20px);
-    border-top: 1px solid transparent;
-    color: govuk-colour("white");
-    height: govuk-spacing(9);
-    padding: govuk-spacing(4);
-    position: relative;
-
-    &:before {
-      border-color: govuk-colour("white");
+    @include focus-not-focus-visible {
+      background: $govuk-brand-colour;
+      border-bottom: govuk-spacing(1) solid $govuk-brand-colour;
+      border-left: none;
+      position: relative;
     }
 
     &:hover {
+      background: govuk-colour("black");
+      border-bottom-color: govuk-colour("mid-grey");
       color: govuk-colour("mid-grey");
-
-      &:after {
-        background-color: govuk-colour("mid-grey");
-      }
-
-      &:before {
-        border-color: govuk-colour("mid-grey");
-      }
     }
 
-    &:focus {
-      color: $govuk-focus-text-colour;
-
-      &:after {
-        background-color: $govuk-focus-text-colour;
-      }
-
-      &:before {
-        border-color: $govuk-focus-text-colour;
-      }
+    @include focus-and-focus-visible {
+      @include govuk-focused-text;
+      border-bottom-color: $govuk-focus-colour;
+      box-shadow: none;
     }
   }
 
+  // Open button modifier
   &.gem-c-layout-super-navigation-header__open-button {
-    border-top-color: govuk-colour("black");
-
-    &:before {
-      transform: translateY(20%) rotate(225deg);
-    }
-
-    @include govuk-media-query($from: "desktop") {
-      background-color: govuk-colour("light-grey");
-      color: $govuk-link-colour;
-
-      &:after {
-        background: $govuk-link-colour;
-      }
-
-      &:before {
-        border-color: $govuk-link-colour;
-      }
-
-      &:focus {
-        background-color: $govuk-focus-colour;
-        color: $govuk-focus-text-colour;
-
-        &:after {
-          background-color: $govuk-focus-text-colour;
-        }
+    @include focus-and-focus-visible {
+      @include govuk-focused-text;
+      border-color: $govuk-focus-colour;
+      box-shadow: none;
+      color: $govuk-focus-colour;
 
+      @include govuk-media-query($from: 360px) {
         &:before {
-          border-color: $govuk-focus-text-colour;
+          @include chevron($govuk-focus-colour, true);
         }
       }
+    }
 
-      &:active {
-        background-color: govuk-colour("light-grey");
-        color: $govuk-link-colour;
-
-        &:before {
-          border-color: $govuk-link-colour;
-        }
-      }
+    @include focus-not-focus-visible {
+      background: govuk-colour("light-grey");
+      border-bottom-color: govuk-colour("light-grey");
+      color: govuk-colour("light-grey");
     }
   }
 }
 
+.gem-c-layout-super-navigation-header__navigation-top-toggle-close-icon {
+  color: $govuk-text-colour;
+  display: none;
+  font-size: 36px;
```

Comment:
Not a comment for this PR really but I wonder if we should feedback to the DS team if they should allow for a version of `govuk-font` which doesn't whack in a bunch of media queries.